examples/toolkit_showcase/toolkit_showcase.ipynb

```diff
@@ -28,47 +28,12 @@
     "</details>\n",
```

Comment:
<p>People will have time to read a bit here - Maybe expand or link to the interchange docs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-toolkit/pull/1368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Done both! I've added an impassioned argument for the OpenFF workflow and bragged about Interchange a bunch.</p>

Reply:
Nice.